jekyll-theme-so-simple.gemspec

```diff
@@ -22,6 +22,6 @@ Gem::Specification.new do |spec|
   spec.add_runtime_dependency "jekyll-feed", "~> 0.1"
   spec.add_runtime_dependency "jekyll-seo-tag", "~> 2.4"
 
-  spec.add_development_dependency "bundler", "~> 2.0.2"
+  spec.add_development_dependency "bundler", "~> 2.1.2"
```

Comment:
Let's remove the version all together... there's no need for it. The following should suffice:

```ruby
spec.add_development_dependency "bundler"
```